pytest_odoo.py

```diff
@@ -211,7 +211,7 @@ def disable_odoo_test_retry():
     try:
         from odoo.tests import BaseCase
         if hasattr(BaseCase, "run"):
-            del BaseCase.run
+            BaseCase.run = lambda *args, **kwargs: None 
     except ImportError:
         pass
```

Comment:
As a simpler, more robust alternative, I would propose:
```python
    try:
        from odoo.tests import BaseCase
        del BaseCase.run
    except (ImportError, AttributeError):
        pass
```
IMO this is also more pythonic.